docs/reference/remap/append.cue

```diff
@@ -0,0 +1,40 @@
+package metadata
```

Comment:
I guess this was accidentally added?

Reply:
No, this is present in every file under `docs/reference`.